libs/ui/src/ui_strings/play_audio_clips.tsx

```diff
@@ -1,23 +1,124 @@
-/* istanbul ignore file - pending implementation. */
-
 import React from 'react';
 
 import { LanguageCode } from '@votingworks/types';
+import { assertDefined } from '@votingworks/basics';
+
+import { useAudioContext } from './audio_context';
+import { AudioPlayer, newAudioPlayer } from './audio_player';
 
 export interface ClipParams {
   audioId: string;
   languageCode: LanguageCode;
 }
 
-export interface PlayAudioClipsProps {
+type PlayAudioClipProps = ClipParams & {
+  onDone: () => void;
+};
+
+function PlayAudioClip(props: PlayAudioClipProps) {
+  const { audioId, languageCode, onDone } = props;
+  const [audioPlayer, setAudioPlayer] = React.useState<AudioPlayer>();
+  const { api, gainDb, playbackRate, webAudioContext } = assertDefined(
+    useAudioContext()
+  );
+
+  const audioOutputRef = React.useRef<AudioPlayer>();
```

Comment:
Is there a particular reason for calling this `audioOutputRef` instead of `audioPlayerRef`?